.github/workflows/ci.yml

```diff
@@ -8,25 +8,32 @@ jobs:
   lint:
     runs-on: 'ubuntu-latest'
     steps:
-      - name: 'Get Changed Files'
-        id: 'files'
-        uses: 'masesgroup/retrieve-changed-files@v3'
-        with:
-          format: 'json'
-        # a force push will cause the action above to fail. Don't do force push when people are
-        # reviewing!
       - name: Check out code.
         uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: 'Get Changed Files'
+        id: 'files'
+        run: |
+          {
+            echo "added_modified<<EOF"
+            git diff --name-only "${BASE_SHA:-$BEFORE_SHA}...${HEAD_SHA:-GITHUB_REF}"
```

Comment:
Would you print the diff result to the output so that we can check it if the CI doesn't work well?

Reply:
Hey, this can be verified by running the github actions in debug mode, then github will log the contents of the env variable. But for sure it would be easier by just printing that in the step. I will amend!